test/jdk/com/sun/crypto/provider/Mac/DigestCloneabilityTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file, including the copyright date and the final newline, should be identical to upstream.

Reply:
Updated